cmd/main.go

```diff
@@ -52,8 +52,10 @@ var (
 	endpoint             = flag.String("endpoint", "unix:/tmp/csi.sock", "CSI endpoint")
 	metricsAddress       = flag.String("metrics-address", "0.0.0.0:9080", "Metrics address")
 	extraVolumeLabelsStr = flag.String("extra-labels", "", "Extra labels to tag all volumes created by driver. It is a comma separated list of key value pairs like '<key1>:<value1>,<key2>:<value2>'.")
-	vendorVersion        string
-	logger               *zap.Logger
+	volumeAttachLimit    = *flag.Int64("volume-attach-limit", 4, "Used to configure Volume attachment limit")
```

Comment:
@shipra101 why we require pointer here ?

Reply:
Since https://pkg.go.dev/flag#Int64 this returns a pointer value. So I have used a dereferencing operator here.